components/lib/badge/Badge.spec.js

```diff
@@ -2,19 +2,17 @@ import '@testing-library/jest-dom';
 import { render } from '@testing-library/react';
 import { Badge } from './Badge';
 
-const snapshot = (props, name) => expect(render(<Badge {...props} />).container).toMatchSnapshot(name);
+import { snapshot } from '../../test';
 
 describe('Badge', () => {
-    test('check snapshots', () => {
-        snapshot({}, 'default');
-        snapshot({ value: 22 }, 'value');
-        snapshot({ size: 'large' }, 'size large');
-        snapshot({ size: 'xlarge' }, 'size xlarge');
-        snapshot({ size: 'invalid' }, 'size invalid');
-        snapshot({ severity: 'success' }, 'severity success');
-        snapshot({ severity: 'info' }, 'severity info');
-        snapshot({ severity: 'warning' }, 'severity warning');
-        snapshot({ severity: 'danger' }, 'severity danger');
-        snapshot({ severity: 'invalid' }, 'severity invalid');
-    });
+    snapshot(<Badge />, 'default');
```

Comment:
To make the snapshot function easier to use across components, it expects the element, not the properties only.